### DISCUSSION TITLE
Very slow image streaming in complex scenes

### DISCUSSION THREAD

**Discussion by djdylex** [microsoft/AirSim#3566] [Support Q&A]
Hi, i have a complicated forest scene I made using various free packages, that included rocks and trees etc. 

![image](https://user-images.githubusercontent.com/6361405/113634804-54985780-9667-11eb-9ff1-76be5fb5b5b5.png)

This doesn't run too slow in the viewport, but when i try to get the images from python, it runs at around ~12 fps. This is very different to the default blocks scene, which gets me ~60fps into python airsim.

I've tried changing the scaleability settings in the viewport. Even on the lowest settings for everything (disabling shadows makes by far the biggest difference), i only get ~20fps.

Is this just a case of having too complex of a scene? Or are there some settings i need to play around with.

I have disabled viewport rendering while flying.

**Comment by hail-linda** [comment]
may be partly solved .
See https://github.com/Cosys-Lab/Cosys-AirSim/issues/82#issuecomment-3187222325

**Reply by jonyMarino** [reply]
Excellent work @hail-linda!
